Sources/Opus/Opus.Custom.swift

```diff
@@ -0,0 +1,95 @@
+import AVFoundation
```

Comment:
What’s the purpose of this file? Can it be deleted?

(I think it should be, if `Opus.Encoder` and `Opus.Decoder` can support nonstandard frame sizes.